### PR TITLE
🌐 Add Spanish translation for `advanced/response-directly.md`

### PR DESCRIPTION
**Pull request by jfunez**

### CODE REVIEW

File: docs/es/docs/advanced/response-directly.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Return a Response Directly
```
Comment (mariacamilagl) [inline docs/es/docs/advanced/response-directly.md:1]:
```suggestion
```

File: docs/es/docs/advanced/response-directly.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+# Retornar Directamente una Respuesta
+
+Cuando tu creas una *operación de camino*, tu puedes retornar normalmente cualquier dato: un `dict`, una `list`, un modelo Pydantic, modelo de base de datos, etc.
```
Comment (mariacamilagl) [inline docs/es/docs/advanced/response-directly.md:4]:
```suggestion
Cuando creas una *path operation*, puedes retornar normalmente cualquier dato: un `dict`, una `list`, un modelo Pydantic, modelo de base de datos, etc.
```

File: docs/es/docs/advanced/response-directly.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+Por defecto, **FastAPI** convertirá automáticamente ese valor de retorno para JSON usando el `jsonable_encoder` explicado en [Codificador Compatible JSON](../tutorial/encoder.md){.internal-link target=_blank}.
+
+Luego, detrás de escena, pondría esos datos compatibles con JSON (por ejemplo, un `dict`) dentro de una `JSONResponse` que se usaría para enviar la respuesta al cliente.
```
Comment (mariacamilagl) [inline docs/es/docs/advanced/response-directly.md:8]:
```suggestion
Luego, detrás de escenas, pondría esos datos compatibles con JSON (por ejemplo, un `dict`) dentro de una `JSONResponse` que se usaría para enviar la respuesta al cliente.
```

File: docs/es/docs/advanced/response-directly.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+Pero tu puedes retornar una `JSONResponse` directamente de tu *operación de camino*.
```
Comment (mariacamilagl) [inline docs/es/docs/advanced/response-directly.md:10]:
```suggestion
Pero puedes retornar una `JSONResponse` directamente de tu *path operation*.
```

File: docs/es/docs/advanced/response-directly.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+Y cuando retornas una `Response`, **FastAPI** la pasará directamente.
+
+No será realizado ninguna conversión de datos con modelos Pydantic, no se convertirán los contenidos de ningún tipo, etc.
+
+Esto te da mucha flexibilidad. Puedes retornar cualquier tipo de datos, sobrescribir cualquer declaración de datos o validación, etc.
+
+## Usando el `jsonable_encoder` en la `Response`
+
+Como **FastAPI** no realiza ningún cambion en la `Response` que tu retornas, debes asegurate que el contenido esta pronto.
```
Comment (mariacamilagl) [inline docs/es/docs/advanced/response-directly.md:29]:
```suggestion
Como **FastAPI** no realiza ningún cambion en la `Response` que tu retornas, debes asegurate que el contenido esta listo para ello.
```

File: docs/es/docs/advanced/response-directly.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+```
+
+!!! note "Detalles Técnicos"
+    También puedes usar `from starlette.responses import JSONResponse`.
+
+    **FastAPI** provee las mismas `starlette.responses` que `fastapi.responses` simplemente como una convención para ti, el desarrollador. Pero la mayoría de las respuestas disponibles vienen directamente de Starlette.
+
+## Retornando una `Response` personalizada
+
+El ejemplo anterior muestra las partes que tu precisas, pero no es muy útil todavía, como tu podrias simplemente retornar el `item` directamente, y **FastAPI** no pondría en una `JSONResponse` para ti, convirtiéndolo en un `dict`, etc. Todo esto por defecto.
```
Comment (mariacamilagl) [inline docs/es/docs/advanced/response-directly.md:46]:
```suggestion
El ejemplo anterior muestra las partes que necesitas, pero no es muy útil todavía, dado que podrías simplemente retornar el `item` directamente, y **FastAPI** lo pondría en una `JSONResponse` para ti, convirtiéndolo en un `dict`, etc. Todo esto por defecto.
```

File: docs/es/docs/advanced/response-directly.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+Ahora, veamos cómo puedes usarlo, para retornar una respuesta personalizada.
```
Comment (mariacamilagl) [inline docs/es/docs/advanced/response-directly.md:48]:
```suggestion
Ahora, veamos cómo puedes usarlo para retornar una respuesta personalizada.
```

File: docs/es/docs/advanced/response-directly.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+Digamos que tu quieres retornar una respuesta <a href="https://en.wikipedia.org/wiki/XML" class="external-link" target="_blank">XML</a>.
```
Comment (mariacamilagl) [inline docs/es/docs/advanced/response-directly.md:50]:
```suggestion
Digamos que quieres retornar una respuesta <a href="https://en.wikipedia.org/wiki/XML" class="external-link" target="_blank">XML</a>.
```

File: docs/es/docs/advanced/response-directly.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+
+Puedes poner tu contenido    XML en una string, ponerla en una `Response` y retornarla:
```
Comment (mariacamilagl) [inline docs/es/docs/advanced/response-directly.md:52]:
```suggestion
Puedes poner tu contenido XML en un string, ponerlo en una `Response` y retornarlo:
```

File: docs/es/docs/advanced/response-directly.md [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+
+```Python hl_lines="1  18"
+{!../../../docs_src/response_directly/tutorial002.py!}
+```
+
+## Notas
+
+Cuando retornas una `Response` directamente, los datos no son validados, convertidos (serializados), ni documentados automáticamente.
+
+Pero todavía es posible documentarlo, como es descrito en [Respuestas Adicionales en OpenAPI](additional-responses.md){.internal-link target=_blank}.
```
Comment (mariacamilagl) [inline docs/es/docs/advanced/response-directly.md:62]:
```suggestion
Pero todavía es posible documentarlo como es descrito en [Respuestas Adicionales en OpenAPI](additional-responses.md){.internal-link target=_blank}.
```

**Review by mariacamilagl — CHANGES_REQUESTED** [review]
Thank you for your work @jfunez 🚀

**Review by mariacamilagl — APPROVED** [review]
Thank you for the translation @jfunez!

**Comment by tiangolo** [comment]
Thanks for the contribution @jfunez! :coffee: 

And thanks @mariacamilagl for the review and fixes! :cake: :nerd_face:

**Pull request merged**